packages/accordions/src/styled/StyledIcon.ts

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled, { css, ThemeProps, DefaultTheme } from 'styled-components';
+import {
+  getColor,
+  getLineHeight,
+  retrieveComponentStyles,
+  DEFAULT_THEME
+} from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'accordions.step_icon';
+
+interface IStyledIcon {
+  isActive?: boolean;
+  isHorizontal?: boolean;
+}
+
+export const StyledIconFlexContainer = styled.div`
+  display: flex;
+  flex-basis: 100%;
+  justify-content: center;
+  width: 100%;
+`;
+
+const sizeStyles = (props: ThemeProps<DefaultTheme>) => {
+  const size = `${props.theme.space.base * 6}px`;
+
+  return css`
+    width: ${size};
+    min-width: ${size};
+    height: ${size};
+    min-height: ${size};
+  `;
+};
+
+export const StyledIcon = styled.div.attrs<IStyledIcon>({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})<IStyledIcon>`
+  display: flex;
+  align-items: center;
+  align-self: ${props => !props.isHorizontal && 'self-start'};
+  justify-content: center;
+  transition: background 0.25s ease-in-out, color 0.25s ease-in-out;
+  /* stylelint-disable-next-line property-no-unknown */
+  margin-${props => (props.theme.rtl ? 'left' : 'right')}: ${props =>
+  !props.isHorizontal && `${props.theme.space.base * 3}px`};
+  margin-bottom: ${props => props.isHorizontal && props.theme.space.base * 2}px;
+  border-radius: 100%;
+  background: ${props =>
+    props.isActive
+      ? getColor('neutralHue', 600, props.theme)
+      : getColor('neutralHue', 200, props.theme)};
+  line-height: ${props => getLineHeight(props.theme.lineHeights.lg, props.theme.fontSizes.sm)};
```

Comment:
Do not use `theme.lineHeights` anywhere except for in `typography`. This should be moved to `sizeStyles` where it can leverage the calculated `size` together with `fontSize` in the `getLineHeight` call.